### PR TITLE
Create afterme.md

### PR DESCRIPTION
**Pull request by EightRice**

### CODE REVIEW

File: projects/afterme.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  <li>Users of traditional centralized estate planning solutions that are motivated by After Me’s lower price point </li>
+
+<li>Post-mortem vloggers that wish to leave behind a record of their personality without being exposed to public scrutiny</li>
+
+<li>Public persons interested in retaining ownership of their natural identity through a claiming service (value prop. on the product roadmap)</li>
+
+
+## Github repo
+<!--Attach a link to your GitHub repo if it's OSS-->
+https://github.com/EightRice/afterme
```
Comment (autonome) [inline projects/afterme.md:31]: Can you license the repo under MIT license?

File: projects/afterme.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+After Me is currently a one-man-show. Andrei Taranu - product manager www.andreitaranu.ml
+
+## How the community can engage
+GitHub Discussion: <!--Start a disucssion with the community here: https://gastmobronsita@gmail.comithub.com/filecoin-project/community/discussions/new and attach the link!-->  
+Email: astmobronsita@gmail.com 
+Twitter:  @andreitaranu
+Discord:  https://discord.gg/RKTHJZF
+
+## How to Contribute
+https://github.com/ipfs/community/discussions/485
```
Comment (autonome) [inline projects/afterme.md:52]: Can you add the discussion link up above?

**Review by autonome — CHANGES_REQUESTED** [review]
Looking good, just need a couple of changes, noted below!

File: projects/afterme.md (hunk 2 above)
Comment (EightRice) [reply projects/afterme.md:52]: I added the link to the project repo's README.md file. Is that what you meant?

**Pull request merged**